src/PSRule.Rules.Azure/rules/Azure.VNET.Rule.ps1

```diff
@@ -302,6 +302,16 @@ Rule 'Azure.VNG.ConnectionName' -Type 'Microsoft.Network/connections' -Tag @{ re
     Match 'Name' '^[A-Za-z0-9]((-|\.)*\w){0,79}$'
 }
 
+# Synopsis: Use availability zone SKU for virtual network gateways deployed with VPN gateway type
+Rule 'Azure.VNG.VPNAvailabilityZoneSKU' -Type 'Microsoft.Network/virtualNetworkGateways' -If { IsVPNGateway } -Tag @{ release = 'GA'; ruleSet = '2021_12'; } {
+    Match 'Properties.sku.name' '^VpnGw[1-5]AZ$';
```

Comment:
Happy to replace regex with something like `$Assert.In()` if you prefer that. Seemed more succint to use regex here.